vortex-expr/src/traversal/references.rs

```diff
@@ -9,6 +9,12 @@ pub struct ReferenceCollector {
     fields: HashSet<FieldName>,
 }
 
+impl Default for ReferenceCollector {
```

Comment:
can you not derive this?

Reply:
sure, no idea why this was added, it was autogen